casper/src/test/scala/coop/rchain/casper/sync/BlockRetrieverRequesAllSpec.scala

```diff
@@ -136,7 +132,7 @@ class BlockRetrieverRequestAllSpec extends AnyFunSpec with BeforeAndAfterEach wi
             blockRetriever.requestAll(timeout).runSyncUnsafe()
             // then
             val Some(requestedAfter) = currentRequests.get.runSyncUnsafe().get(hash)
-            requestedAfter.timestamp shouldBe time.clock
+            requestedAfter.timestamp shouldBe 0
```

Comment:
Why timestamp is 0 here?

Reply:
With TestScheduler you are in control of the _tick_ of the Timer so here we don't even need to do anything because time will stay at zero. It can be improved to make it more precise.